web/modules/custom/dpl_login/dpl_login.module

```diff
@@ -27,21 +29,26 @@ use function Safe\sprintf as sprintf;
  *   User info from external service.
  * @param mixed[] $context
  *   Various openid_connect context. Tokens etc.
+ *
+ * @throws Exception
  */
 function dpl_login_openid_connect_userinfo_alter(array &$userinfo, array $context): void {
 
-  // If we cannot resolve uinque id we cannot continue.
-  if (!$id = $userinfo['attributes']['uniqueId'] ?? FALSE) {
-    $userinfo = [];
-    return;
+  // If we cannot resolve cpr we cannot continue.
```

Comment:
I guess it should be:
```
// If we cannot resolve both cpr and uniqueId we cannot continue.

Reply:
Yes, you are right. 
I have updated the comment now.